packages/react/label/src/Label.tsx

```diff
@@ -80,17 +80,16 @@ Label.displayName = NAME;
 
 /* -----------------------------------------------------------------------------------------------*/
 
-const useLabelContext = <E extends HTMLElement>(ref?: React.RefObject<E>) => {
+const useLabelContext = <E extends HTMLElement>(element?: E | null) => {
   const context = React.useContext(LabelContext);
 
   React.useEffect(() => {
     const label = context?.ref.current;
```

Comment:
What about this ref? does it need to be reactive too?

Reply:
I don't think so because the consumer has to be inside the label for the Context to exist. So the `label` will always exist. We create this hook alongside the label component which we know isn't conditionally rendered or anything.